src/libsyncengine/jobs/network/API_v2/abstracttokennetworkjob.cpp

```diff
@@ -97,6 +97,18 @@ void AbstractTokenNetworkJob::updateLoginByUserDbId(const Login &login, int user
     }
 }
 
+void AbstractTokenNetworkJob::clearCacheForUser(int userDbId) {
```

Comment:
As the intent seems to avoid exception, let's declare these method `no except`.
It might be useful to:
- `assert(false)` if the condition is not `true`,
- or log something as an error.

Reply:
Done

Reply:
Done